code/model/editableformfields/EditableMultipleOptionField.php

```diff
@@ -171,7 +177,7 @@ public function getHasAddableOptions() {
 	 */
 	protected function getOptionsMap() {
 		$optionSet = $this->Options();
-		$optionMap = $optionSet->map('EscapedTitle', 'Title');
+		$optionMap = $optionSet->map('Value', 'Title');
```

Comment:
Can you please add a fallback for value so that, if it's left empty, it defaults to EscapedTitle? Otherwise people upgrading to this version will suddenly get broken options. :D